lib/rules/disallow-space-after-object-keys.js

```diff
@@ -134,7 +134,7 @@ module.exports.prototype = {
             var maxKeyEndPos = 0;
             var tokens = [];
             node.properties.forEach(function(property) {
-                if (property.shorthand || property.kind !== 'init') {
+                if (property.shorthand || property.kind !== 'init' || property._babelType === 'SpreadProperty') {
```

Comment:
So what happened with eslint was they weren't supporting anything not in spec so we have babel-eslint and also eslint-plugin-babel which rewrites eslint rules to incorporate es7 stuff like decorators etc. However in ESLint 1.0 (http://eslint.org/blog/2015/07/eslint-1.0.0-released/#new-language-features) `ExperimentalRestProperty` and `ExperimentalSpreadProperty` which are both es7 are supported in espree because of all the react users so they wouldn't need to use babel-eslint.